infrastructure/database/templates/07_create_or_update_views.template.sql

```diff
@@ -22,25 +22,105 @@ FROM public.radolan;
 
 CREATE MATERIALIZED VIEW public.rainfall AS
 SELECT public.tree_radolan_tile.tree_id, tile_rainfall.rainfall_mm_14d_sum as rainfall_in_mm
-FROM public.tree_radolan_tile 
-JOIN 
+FROM public.tree_radolan_tile
+JOIN
 	(SELECT DISTINCT ON (tile_id)
 	tile_id, rainfall_mm_14d_sum
 	FROM public.radolan_14d_agg
 	ORDER BY tile_id, timestamp DESC) as tile_rainfall
 ON public.tree_radolan_tile.tile_id = tile_rainfall.tile_id;
 
-CREATE OR REPLACE VIEW private.training_data AS
-SELECT sensor_measurements.tree_id, sensor_measurements.sensor_id, sensor_measurements.timestamp, sensor_measurements.value,
-		shading.winter, shading.spring, shading.summer, shading.fall,
-		trees.gattung_deutsch, trees.standalter,
-		weather_14d_agg.temp_avg_c_14d_avg, weather_14d_agg.wind_avg_ms_14d_avg, weather_14d_agg.temp_max_c_14d_avg, weather_14d_agg.wind_max_ms_14d_avg 
-FROM private.sensor_measurements
-LEFT JOIN public.shading ON public.shading.tree_id = sensor_measurements.tree_id
-LEFT JOIN public.trees ON trees.id = sensor_measurements.tree_id
-LEFT JOIN public.weather_14d_agg ON sensor_measurements.timestamp = weather_14d_agg.timestamp
-LEFT JOIN public.radolan_14d_agg ON sensor_measurements.timestamp = radolan_14d_agg.timestamp
-ORDER BY timestamp DESC;
+CREATE MATERIALIZED VIEW public.vector_tiles AS
+SELECT
+	trees.id AS trees_id,
+	trees.standortnr AS trees_standortnr,
+	trees.kennzeich AS trees_kennzeich,
+	trees.namenr AS trees_namenr,
+	trees.art_dtsch AS trees_art_dtsch,
+	trees.art_bot AS trees_art_bot,
+	trees.gattung_deutsch AS trees_gattung_deutsch,
+	trees.gattung AS trees_gattung,
+	trees.strname AS trees_strname,
+	trees.hausnr AS trees_hausnr,
+	trees.pflanzjahr AS trees_pflanzjahr,
+	trees.standalter AS trees_standalter,
+	trees.stammumfg AS trees_stammumfg,
+	trees.baumhoehe AS trees_baumhoehe,
+	trees.bezirk AS trees_bezirk,
+	trees.eigentuemer AS trees_eigentuemer,
+	trees.zusatz AS trees_zusatz,
+	trees.kronedurch AS trees_kronedurch,
+	trees.geometry AS trees_geometry,
+	trees.lat AS trees_lat,
+	trees.lng AS trees_lng,
+	trees.created_at AS trees_created_at,
+	trees.updated_at AS trees_updated_at,
+	trees.street_tree AS trees_street_tree,
```

Comment:
should it also include Baumscheibe from latest merge in main?

Reply:
Sure. I'll push a fix

Reply:
Cant see baumscheibe in the current staging env.

Reply:
It's in main with this PR: https://github.com/technologiestiftung/qtrees-ai-data/pull/23

Reply:
Ah. Okay. Are you doing PRs directly into main? This one goes into staging

Reply:
Oh. Okay, maybe I missed that. Didn't know about staging.

Reply:
Nico hat zuletzt aber auch nach main gemerged. Hatten wir da mal ne andere Konvention besprochen?

Reply:
I don't know. I just saw the staging branch and thought: "hey lets use staging"